docs/tools/vdb_table/data/nucliadb.json

```diff
@@ -26,127 +26,127 @@
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 497,
-  "vector_launch_year": 0,
+  "github_stars": 537,
+  "vector_launch_year": 2021,
   "metadata_filter": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "facets": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_vec": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/README.md",
     "comment": ""
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "full_text": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/nucliadb_node/README.md#nucliadb_paragraphs-and-nucliadb_fields",
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/nucliadb_node/README.md#nucliadb_paragraphs-and-nucliadb_fields",
```

Comment:
this doesn't seem like the right link. Please provide a link to where we are able to provide the model name (something like a Sentence Transformers model), and have Nuclia compute embeddings for a text column with it